### DISCUSSION TITLE
Ios trigger

### DISCUSSION THREAD

**Discussion by xyzmauro** [indomus/forum#350] [HUB: Home Assistant]
Ciao a tutti, avrei necessità di far scattare un trigger su HA quando posticipo oppure spengo la sveglia su iPhone ( iOS 12.5.3 ), ho installato Companion app convinto che esponesse il dato ma nulla ( so quanti km faccio al giorno e i piani saliti e discesi, ma la sveglia no 😕 ) qualcuno ha provato/trovato una soluzione?

grazie in anticipo

**Comment by gforment** [comment]
Puoi provare a giocare con il valore di quanto manca all'allarme successivo (se cerchi in rete "Minutes until Next Alarm" con "home assistant" trovi come fare): se spegni un allarme probabilmente il suo valore aumenta di parecchio e da lì puoi capire (magari incrociandolo con l'ora in cui succede) che l'allarme è stato spento. Altrimenti qualche task di Tasker (ma solo Android) integrato con gli eventi dell'allarme che poi triggera HA.
Mi vengono in mente questi modi

**Comment by xyzmauro** [comment]
Per ora sono riuscito da iPad con l’app Comandi ma ha bisogno di iOS 13 come minimo quindi da iPhone per ora nulla
